docs/pages/guides/tree-shaking.mdx

```diff
@@ -94,13 +90,15 @@ module.exports = function (api) {
 };
 ```
 
-</Tab>
-
-</Tabs>
+</Collapsible>
 
 ## Remove development-only code
 
-In your project, there might be code designed help with the development process. It should be excluded from the production bundle. To handle these scenarios, use the `process.env.NODE_ENV `environment variable or the non-standard `__DEV__` global boolean. For example:
+In your project, there might be code designed help with the development process. It should be excluded from the production bundle. To handle these scenarios, use the `process.env.NODE_ENV `environment variable or the non-standard `__DEV__` global boolean.
```

Comment:
```suggestion
In your project, there might be code designed to help with the development process. It should be excluded from the production bundle. To handle these scenarios, use the `process.env.NODE_ENV `environment variable or the non-standard `__DEV__` global boolean.
```

There is a typo here in first line from my previous PR suggestions that I missed.